src/components/DataTable.js

```diff
@@ -11,22 +11,18 @@ export default class DataTable extends React.Component {
   }
 
   componentDidMount() {
-    this.setWide();
     window.addEventListener('resize', this.setWide.bind(this));
   }
 
-  componentWillUnmount() {
-    window.removeEventListener('resize', this.setWide);
-  }
-
-  componentDidRecieveProps() {
+  propsDidChange() {
```

Comment:
should this be componentWillReceiveProps()? https://facebook.github.io/react/docs/component-specs.html#lifecycle-methods